### PR TITLE
feat: [CI-4886]: Issue templates for community issues

### PR DESCRIPTION
**Pull request by udayvunnam**
**Summary**

Added default issue templates that ask for better details that avoid issue creations like https://github.com/harness/harness-core-ui/issues/9463 from the community

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by udayvunnam** [comment]
retrigger jest

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (abhinavrastogi-harness) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: we can remove this section since we dont support mobile devices at all

**Review by abhinavrastogi-harness — COMMENTED** [review]
awesome!

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**